### PR TITLE
feat: optionally hide chat during screenshots

### LINKED ISSUE #206: Hide private chat on screenshot frame
Private chat should be hidden when we take the screenshot to not leak PMs.

Options I'd consider:
 - switching chat to `Game` for a single frame and hiding split chat
 - hiding the chat widget for a frame and hiding split chat

Potential pitfalls:
 - split private chat option
 - level up dialog is in the same place the chat normally is, so blacking out the region isn't really an option

### PR DESCRIPTION
**Pull request by iProdigy**
Closes #206
Related: https://github.com/MidgetJake/UniversalDiscordNotifier/issues/36

### CODE REVIEW

File: src/main/java/dinkplugin/DinkPluginConfig.java [hunk 1]
```diff
@@ -238,6 +238,17 @@ default PlayerLookupService playerLookupService() {
...
 
+    @ConfigItem(
+        keyName = "screenshotHideChat",
+        name = "Hide Chat in Images",
+        description = "Whether to hide the chat box and private messages when capturing screenshots",
+        position = 1009,
+        section = advancedSection
+    )
+    default boolean screenshotHideChat() {
+        return false;
```
[the remark this answers is not in the document]
Comment (iProdigy) [reply src/main/java/dinkplugin/DinkPluginConfig.java:250]: Leaving this as `false` for now after talking to Felanbird/pajlada; Folks who have already enabled screenshots are implicitly indicating that their privacy desire isn't dominating & the current implementation induces a noticeable [flicker](https://gachi.gay/Zhlbd) that may be strange to enforce as a default (personally I'd use this feature despite the flicker, but that may not go for everyone)

**Review by pajlada — APPROVED** [review]
Code looks good, I think the default value for this setting is good - only nitpick I can think of is that we don't mention that we also hide the private messages above chat in the changelog, but maybe that is already encompassed by "chat"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+- Minor: Add advanced setting to hide chat when screenshotting. (#208)
```
Comment (iProdigy) [inline CHANGELOG.md:3]:
```suggestion
- Minor: Add advanced setting to hide chat (and PMs above chat) when screenshotting. (#208)
```

**Review by Felanbird — APPROVED** [review]
Seems to work as expected, only tested level-ups & quests for now.

File: CHANGELOG.md (hunk 2 above)
Comment (Felanbird) [reply CHANGELOG.md:3]: meh - if they don't want PMs hidden also, it's just as simple as hiding the widget manually, and not using the setting.

**Pull request merged**